### DISCUSSION TITLE
WALC settings icon not there

### DISCUSSION THREAD

**Discussion by ignacionelson** [cstayyab/WALC#236] [Troubleshooting] [answered]
Hi! I'm on the latest version, on Linux Mint 21.3. 
Is there a keyboard shortcut to access walc's settings? The icon is not there on the profile bar.
Thanks

**Comment by Sadi58** [comment, marked as answer]
This seems to be a recurring problem; the settings icon next to the user's profile picture in the toolbar of the sidebar disappears every time something is broken. See: https://github.com/WAClient/WALC/issues/158
That's why I've switched to https://github.com/keshavbhatt/whatsie

**Comment by ignacionelson** [comment]
Thank you Sadi
